torchmetrics/functional/__init__.py

```diff
@@ -33,7 +33,6 @@
 from torchmetrics.functional.classification.specificity import specificity  # noqa: F401
 from torchmetrics.functional.classification.stat_scores import stat_scores  # noqa: F401
 from torchmetrics.functional.image_gradients import image_gradients  # noqa: F401
-from torchmetrics.functional.nlp import bleu_score  # noqa: F401
```

Comment:
@Borda should we deprecate this?
Only influences people using the full import path
`from torchmetrics.functional.nlp import bleu_score`
whereas
`from torchmetrics.functional import bleu_score`
will still work

Reply:
yep, rather make simple deprecation loop